src/DataFixtures/SpecialBenefitFixtures.php

```diff
@@ -20,12 +20,13 @@ public function __construct(SpecialBenefitManagerInterface $specialBenefitManage
 
     /**
      * @param ObjectManager $manager
+     *
      * @throws \Exception
      */
-    public function load(ObjectManager $manager)
+    public function load(ObjectManager $manager): void
     {
         $faker = Faker::create();
-        for ($specialBenefitNbr = 0; $specialBenefitNbr < 10; ++$specialBenefitNbr) {
+        for ($specialBenefitNbr = 0; $specialBenefitNbr < 10; $specialBenefitNbr++) {
```

Comment:
Beware, changing from pre to post incrementation will result in naming from 0 to x instead of 1 to x+1

Reply:
This change doesn't affect anything :) The
 last statement of a "for" loop is always executed at the end ++$var and $var++ will have the same effect here

https://3v4l.org/DBTRN

Reply:
Weird, thanks for the info 👍 
So that's pre/post incrementing at the end of the loop them...?